css/components/avatar.css

```diff
@@ -29,6 +28,8 @@
 
 .avatar > img {
```

Comment:
try not to use an element selector even as a selective child.
why? - because the user may use SVG instead.
solution - use a universal selector as combinator (according to your HTML structure)

Reply:
Thanks for the suggestions, made the suggested changes!